changelog/@unreleased/pr-6127.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: improvement
+improvement:
+  description: '[TTS] Force enable sweep queue writes.'
```

Comment:
I think we probably want a louder, clearer message here (given this may be interesting for people on support who run into issues!)